### PR TITLE
Remove curly brace initialization to support VS2012

### PR DESCRIPTION
**Pull request by usakhelo**

### CODE REVIEW

File: src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp [hunk 1]
```diff
@@ -78,6 +78,12 @@ namespace
     {
         std::string                     m_name;
         renderer::ObjectInstance::Side  m_side;
+
+        explicit MaterialSlot(const std::string name, const renderer::ObjectInstance::Side side)
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp]: I'm surprised that this compiles without a default constructor, but if it does, then fine :)

Comment (usakhelo) [reply src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp]: Hi @dictoon , 
Yes it does. Do you want me to rebase this PR?

Comment (usakhelo) [reply src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp]: I checked recent commits and it looks like we are going to drop 2012. Do you want me to delete this PR instead?

Comment (dictoon) [reply src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp]: Yes, it's not longer needed. Thanks for your work though!

**Comment by dictoon** [comment]
VS 2012 support dropped.

**Pull request closed without merging**